### DISCUSSION TITLE
Badge with github gist stars

### DISCUSSION THREAD

**Discussion by maratori** [badges/shields#8264] [Q&A] [answered]
I can easily create a badge with number of stars for github repo.

But how to do the same for github gist?
For example [this gist](https://gist.github.com/maratori/47a4d00457a92aa426dbd48a18776322) has 11 stars for now.

Maybe related to #7884

**UPD:** I've created issue #8269

**Comment by calebcartwright** [comment]
To the best of my knowledge, we do not currently provide any badges for GitHub gists, which require utilizing a different API surface than what we use to get the required data for the badges we provide for GitHub repositories.

It would be a reasonable badge for us to support, so if it's something you're interested in I'd recommend opening a new badge request with the relevant data points: https://github.com/badges/shields/issues/new?assignees=&labels=service-badge&template=3_Badge_request.md

**Reply by maratori** [reply, marked as answer]
Thanks, I've created issue #8269

**Comment by chris48s** [comment]
Note: This one would have to use the graphql api. The Rest endpoint for gists doesn't show stars
https://docs.github.com/en/rest/gists/gists#get-a-gist
but you can query it with graphql
https://docs.github.com/en/graphql/reference/objects#gist

**Reply by maratori** [reply]
Thanks
